tasks/qiime-import/mypy.ini

```diff
@@ -17,3 +17,12 @@ disallow_incomplete_defs = True
 no_implicit_optional = True
 strict_optional = True
 allow_redefinition = False
+
+
+# Per-module options:
+# https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
+[mypy-cchardet.*]
+ignore_missing_imports = True
+
+[mypy-pandas.*]
+ignore_missing_imports = True
```

Comment:
pandas has package called `pandas-stubs` which gets installed by running `mypy --install-types` so it should not be present in this file